packages/loader/container-loader/src/container.ts

```diff
@@ -721,6 +721,7 @@ export class Container
 				},
 				error,
 			);
+			this.close(normalizeError(error));
```

Comment:
To be honest, I do not remember all the history (would need to look it up), but I think we went back and forth on that in the past. Maybe @markfields remembers / has any insights or suggestions?
Looking at the code, I'm actually a bit suprised to see couple cases in ContainerRuntime where compiler does not force us to do wrapError() or normilizeError() - we pass any and it compiles (but does not here).
Maybe intention was to provide extra properties to indicate callsite (in case we have no stack)?

Let's look at it holistically, not as part of this PR :)